runtime/doc/news.txt

```diff
@@ -358,6 +358,8 @@ TREESITTER
   descendant itself.
 • |LanguageTree:parse()| optionally supports asynchronous invocation, which is
   activated by passing the `on_parse` callback parameter.
+• |vim.treesitter.query.set()| can now inherit and/or extend runtime file
+  queries in addition to overriding.
```

Comment:
Is there a small practical example we can show here? E.g. mentioning 'spell' explicitly would be very helpful.

Reply:
Good idea. I'll add the example to `treesitter.txt` though.